src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ListAssistHearingMessageHelperTest.java

```diff
@@ -44,6 +45,7 @@ public void shouldSendExpectedCancellationMessage() {
         assertEquals("1234", actualRequest.getCcdCaseId());
         assertEquals(LIST_ASSIST, actualRequest.getHearingRoute());
         assertEquals(CANCEL_HEARING, actualRequest.getHearingState());
+        assertEquals(CancellationReason.OTHER, actualRequest.getCancellationReason());
```

Comment:
I appreciate you're following existing patterns here, but worth noting that use of Hamcrest is better these days

Reply:
Used the hamcrest for the line in the test I added